docs/en/Running-Alluxio-on-EC2-Yarn.md

```diff
@@ -121,13 +121,13 @@ Note that adding `-DskipTests -Dfindbugs.skip -Dmaven.javadoc.skip -Dcheckstyle.
 but it makes the build run significantly faster.
 
 To customize Alluxio master and worker with specific properties (e.g., tiered storage setup on each
-worker), one can refer to [Configuration settings](Configuration-Settings.html) for more
-information. To ensure your configuration can be read by both the ApplicationMaster and Alluxio
-master/workers, put `alluxio-site.properties` under `~/.alluxio/` on each EC2 machine.
+worker), see [Configuration settings](Configuration-Settings.html). To ensure your configuration can be
+read by both the ApplicationMaster and Alluxio master/workers, put `alluxio-site.properties` under
+`~/.alluxio/` on each EC2 machine.
```

Comment:
Does this need to be in the home folder of the user the Alluxio process / AM is running under?